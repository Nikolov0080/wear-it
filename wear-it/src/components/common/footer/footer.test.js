import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './footer';
// import { BrowserRouter } from 'react-router-dom';

it('renders correctly Footer', () => {
    const tree = renderer
        .create(<Footer  />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});