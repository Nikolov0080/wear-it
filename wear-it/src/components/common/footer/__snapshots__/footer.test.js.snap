// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Footer 1`] = `
<footer
  className="footer"
>
  <div
    className="brand"
  >
    wear it
  </div>
  <hr />
  <div
    className="copyR"
  >
    <span>
      Copyright Â© All Rights Received
    </span>
  </div>
</footer>
`;
